src/database/gravity-db.c

```diff
@@ -733,7 +748,7 @@ bool in_gravity(const char *domain, const int clientID, clientsData* client)
 	// Update statement if has just been initialized
 	if(stmt == NULL)
 	{
-		stmt = whitelist_stmt->get(whitelist_stmt, clientID);
```

Comment:
This was incorrect and gets fixed here as well. It is fallback code which is likely why we have never seen reports for this. Same below.